frontend/src/layouts/widget/UserMenu.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import {
+  UncontrolledDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+} from "reactstrap";
+import { FiLogOut } from "react-icons/fi";
+import { IoMdSettings } from "react-icons/io";
+
+import { UserBubble, DropdownNavLink } from "@certego/certego-ui";
+
+import { useAuthStore } from "../../stores";
+
+const djangoAdminLink = (
+    <>
+      <DropdownNavLink to="/admin/" target="_blank">
+        <IoMdSettings className="me-2" /> Django Admin Interface
+      </DropdownNavLink>
+      <DropdownItem divider />
+    </>
+);
+
+function UserMenu(props) {
+    // auth store
+    const [user, isSuperuser] = useAuthStore(
+      React.useCallback((s) => [s.user, s.isSuperuser], [])
+    );
+
+    return (
+        <UncontrolledDropdown nav inNavbar {...props}>
+          <DropdownToggle nav className="text-center">
+            <UserBubble size="sm" userInfo={user} />
+          </DropdownToggle>
+          <DropdownMenu end className="bg-dark" data-bs-popper>
+            <DropdownItem text>
+              logged in as <b>{`${user?.username}`}</b>
+            </DropdownItem>
+            <DropdownItem divider />
+            {/* Django Admin Interface */}
+            {isSuperuser ? djangoAdminLink : null}
```

Comment:
In this case we can use another operator instead of ternary operator: `condition && <to_render_if_true>`.